utils.py

```diff
@@ -168,7 +168,39 @@ def __init__(self, name, parser, handler, args=[], kwargs={}, default_feed='full
         self.args = args
         self.kwargs = kwargs
 
+    def metadata(self, request):
+        meta = LazyBuilder(version='0.1.0')
```

Comment:
Should we use the actual version of the parser, which leaves of the version tag in the xml in most cases: `LazyBuilder(version=self.parser.version)`? Or does this only work when a version is present?